### ISSUE TITLE
Link Login Modal to Help Answer future questions

### ISSUE THREAD

**Issue by kevinzluo** [publiclab/plots2#4231]
We are creating a login/signup modal, next step is to link to various pages. In this issue we will be linking it to request an activity (The subscription link here). For modularity, this is separate from linking the button.
![image](https://user-images.githubusercontent.com/44309027/49712758-891cb580-fc0b-11e8-8dc9-0e78952a8d71.png)

**Comment by oorjitchowdhary** [comment]
@kevinzluo Are you working on this one?

**Comment by okonek** [comment]
@SidharthBansal Can I do it now?

**Comment by SidharthBansal** [comment]
@oorjitchowdhary are you going to create a PR for this? I think this is one of the many issues claimed by you.
Thanks.

**Comment by oorjitchowdhary** [comment]
https://github.com/publiclab/plots2/blob/70fdeb1a5fb43d33aea71868d6d346c39959ad0b/app/views/grids/_notes.html.erb#L66-L73

The `requireLogin` class is already implemented here by @kevinzluo ... I think this might have also solved this issue.. But we may need to try it out on localhost.

**Comment by oorjitchowdhary** [comment]
@SidharthBansal How can I add the grid in the wiki page.. I'm not able to do it. 
was it something like `[grid:test}`?

**Comment by SidharthBansal** [comment]
I also think that this has solved this issue.
Can you please try out if issue is still present? In case it is not present then please close the issue.
Thanks Oorjit.

**Comment by oorjitchowdhary** [comment]
I'm trying it out but I'm unable to add the grid to on localhost.. I'm not able to remember the command. How do i add the grid

**Comment by SidharthBansal** [comment]
I will suggest you to read following first then do other things.
https://publiclab.org/wiki/power-tags this will help you understanding one
of the extra ordinary feature built by @jywarren. Then please continue with
the work. I believe your doubt will be solved after reading it up.

**Comment by oorjitchowdhary** [comment]
Hi, @SidharthBansal .. This is not yet solved. I tried it. I was able to link the modal here but the current flow is:
1. User clicks on help answer future questions
2. Login modal shows up
3. A successful sign in redirects to the same page
Is this flow okay?

I had reported a similar issue #4218 which I'm currently trying to complete.
I have the same error there.
Through, the `class='requireLogin'`, I'm  able to display the login modal but it doesn't redirect to the button's command in both the issues #4218 and #4231
